docs/releases/3.2.txt

```diff
@@ -223,6 +223,10 @@ Models
 * The new :attr:`.UniqueConstraint.opclasses` attribute allows setting
   PostgreSQL operator classes.
 
+* Inherited fields compare unequal when accessed from different models.
```

Comment:
Consider rephrasing this (somehow) to specifically/explicitly make it clear that this is talking about comparison of model Field instances only? The ambiguity currently _could_ suggest that `instance_a.text == instance_b.other_text` would always be not-equal.

Reply:
Thanks for the suggestion. I've updated the wording.